docs/questions-and-answers.md

```diff
@@ -10,10 +12,76 @@ An Agent Service is an Autonomous Service which is implemented as a multi-agent
+<li>The keeper collects all the signatures and sends the transaction onchain.</li>
+<li>All agents wait for the transaction to be mined and validate the output.</li>
+<li>Done</li>
+</ol>
+</details>
+
+## Security
+
+<details><summary>How are Autonolas services run?</summary>
+
+Autonolas services are composed of multiple agents that run the same code and agree on its output. These agents are executed in different nodes that are run by independent operators. In order to avoid centralizing this power, Autonolas does not play a role here so it does not offer a cloud platform. Each operator must select and setup the infrastructure that best suits their needs.</details>
+
+<details><summary>What happens if my node is hacked?</summary>
+
+As in any other online service, Autonolas nodes are exposed to the risk of being breached. At the individual level, Autonolas does not provide a solution to this and it’s up to the node operator to keep the node safe. At the service level, on the other hand, services are secured in three ways:</p>
+
+<ul>
+<li>Each agent service implements a custom protocol that expects very narrow message flow, so a hypothetical node running malicious code would need to express its intentions within this protocol, otherwise the other agents will ignore its messages.</li>
+
+<li>Even in the case of an agent sending valid, malicious messages to the service, the decentralized nature of Autonolas services means that the majority threshold of agents (2/3 + 1) should agree before committing a malicious transaction, so it is not enough to breach a bunch of nodes.</li>
+
+<li>Services are crypto-economically secured: agents are incentivised to behave honestly by the fact that certain misbehavior can be detected and punished, so it is economically not profitable to cheat.</li>
+</ul>
+</details>
+
+<details><summary>For critical operations, like sending a transaction to a blockchain, it is not enough with trusting that the agents will behave honestly, and further security and cryptographic mechanisms are required. Does the framework provide those mechanisms?</summary>
+For sending transactions to a chain, for example, a multisig approach is used (currently Gnosis Safe) so that a threshold of agents must always approve and validate operations.
+</details>
+
+## Costs
+
+<details><summary>How much does it cost to run an autonomous service using the framework?</summary>
+We don’t define what an agent service does and how it is configured (e.g. number of agents in it), so the costs are subjective to the service. At the very minimum there will be infrastructure costs.
+On top of that, if a service sends transactions to a chain, it will incur in fee costs that will depend on the selected chain. As an example, for a simple service of four agents that make a simple contract call every five minutes, a monthly cost of $3000 in Ethereum and $1.5 in Polygon is estimated, but this number will wildly vary depending on gas costs.
```

Comment:
```suggestion
We do not define what an agent service does and how it is configured (e.g. number of agents in it), therefore the costs are subjective to the service. At the very minimum there will be the costs of running the agent on cloud or local infrastructure.
On top of that, if a service sends transactions to a chain, it will incur in fee costs that will depend on the selected chain. As an example, for a simple service of four agents that makes a simple contract call every five minutes, a monthly cost of $3000 in Ethereum and $1.5 in Polygon is presently estimated (at gas costs of X), but this number will wildly vary depending on gas costs.
```